flink-core/src/main/java/org/apache/flink/configuration/BlobServerOptions.java

```diff
@@ -118,4 +118,10 @@
             key("blob.client.connect.timeout")
                     .defaultValue(0)
                     .withDescription("The connection timeout in milliseconds for the blob client.");
+
+    /** The maximum size limit of blob cache on TaskExecutor. */
+    public static final ConfigOption<Long> BLOB_CACHE_SIZE_LIMIT =
+            key("blob.cache.size.limit")
+                    .defaultValue(1_024 * 1_024 * 1_024 * 10L)
+                    .withDescription("The maximum size limit of blob cache for each TaskExecutor.");
```

Comment:
the description needs clearly state how it affects permanent blob and transient blob size.

Reply:
> the description needs to clearly state how it affects permanent blob and transient blob size limit.

I've updated the descriptions here.

Reply:
This option has been removed.